src/modules/article.js

```diff
@@ -24,20 +24,72 @@ const createArticle = async (title, body) => {
     const response = await axios.post(
       "/admin/articles",
       {
-        article: 
-        {
+        article: {
           title: title,
           body: body
         }
       },
       {
         headers: headers
       }
-      );
+    );
     return response;
   } catch (error) {
-    return error.response
+    return error.response;
   }
 };
 
-export { getCurrentArticle, getArticles, createArticle };
+const getUnpublishedArticles = async () => {
+  let headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"));
+  try {
+    let response = await axios.get(`/admin/articles`, {
+      headers: headers
+    });
+    return response.data.articles;
+  } catch (error) {
+    return error.message;
+  }
+};
+
+const publishArticle = async id => {
+  let headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"));
+  try {
+    let response = await axios({
+      url: `/admin/articles/${id}`,
+      headers: headers,
+      method: "put",
+      params: {
+          published: true
+      }
+    });
+    return response.data
+  } catch (error) {
+    return error.response.data.errors;
+  }
+};
+
+const undoPublishArticle = async id => {
+  let headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"));
+  try {
+    let response = await axios({
+      url: `/admin/articles/${id}`,
+      headers: headers,
+      method: "put",
+      params: {
+          published: false
+      }
```

Comment:
should also set publisher_id to null(nil?) as it's set to the current_user.id by the backend

Reply:
Fixed this in the backend